changelogs/unreleased/6523-therealak12-small.md

```diff
@@ -0,0 +1 @@
+- Improve response time by utilizing a query param (`filter=http\..*\.downstream_cx_active`) when querying `/stats/prometheus` for `envoy_http_downstream_cx_active` 
```

Comment:
nit: lets mention that this is a shutdown manager change here, also can remove the leading `- `

Reply:
```suggestion
Improve shutdown manager query to the Envoy stats endpoint for active connections by utilizing a regex filter query param.
```

Reply:
Thanks. I've applied your suggestion.